docs/src/examples/components/Animation/Playground.tsx

```diff
@@ -0,0 +1,2 @@
+// Playground is not supported by this component
+export default null
```

Comment:
So about this one, can we export some component like `<AutogeneratedPlayground />` that will trigger that we want auto generated playground for the component, and skip it for the components that we don't want to have a playground?

Reply:
We agreed to have a blacklist separately